manifests/f/Freeplane/Freeplane/1.9.3/Freeplane.Freeplane.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate
+
+PackageIdentifier: Freeplane.Freeplane
+PackageVersion: 1.9.3
+PackageLocale: en-US
+Publisher: The Freeplane team

```

Comment:
```suggestion
Publisher: Open source
```

Reply:
Is this needed for proper correlation with Add/Remove program entries?

Reply:
Yes.